### PR TITLE
Fix contour close

### PR DESCRIPTION
**Pull request by Yaraslaut**
When closing from window manager GUI will disappear but process was still alive, this PR fix this behavior

### CODE REVIEW

File: src/contour/TerminalSession.cpp [hunk 1]
```diff
@@ -478,8 +478,9 @@ void TerminalSession::onClosed()
     auto const now = steady_clock::now();
     auto const diff = std::chrono::duration_cast<std::chrono::seconds>(now - _startTime);
 
-    if (auto const* localProcess = dynamic_cast<terminal::Process const*>(&_terminal.device()))
+    if (auto*  localProcess = dynamic_cast<terminal::Process *>(&_terminal.device()))
```
Comment (christianparpart) [inline src/contour/TerminalSession.cpp:481]:
```suggestion
    if (auto* localProcess = dynamic_cast<terminal::Process *>(&_terminal.device()))
```

**Review by christianparpart — COMMENTED** [review]
I'm surprised that clang-format didn't catch this? 🤔

File: src/contour/TerminalSession.cpp (hunk 1 above)
Comment (Yaraslaut) [reply src/contour/TerminalSession.cpp:481]: xD ye, clang format shows weird behavior lately

**Review by christianparpart — APPROVED** [review]
Nice. Many thanks. :)

**Pull request merged**